git_machete/annotation.py

```diff
@@ -13,8 +13,10 @@ def __init__(self, annotation: str):
         self._annotation_without_qualifiers = annotation
         self._rebase_text = ''
         self._push_text = ''
+        self._slide_out_text = ''
         self.rebase = True
         self.push = True
+        self.slide_out = None
 
         match_pattern: Callable[[str], str] = lambda text: f'.*\\b{text}=no\\b.*'
         sub_pattern: Callable[[str], str] = lambda text: f'[ ]?{text}=no[ ]?'
```

Comment:
Nit: maybe just nested defs instead of lambdas? easier to read

Reply:
Yeah, i was 50/50 on this one, decided that it's nicer to have one liners than defs but will change